database/tables/alternative_groups.pg

```diff
@@ -3,6 +3,7 @@ columns
     id: bigint not null default nextval('alternative_groups_id_seq'::regclass)
     number: integer
     number_choose: integer
+    question_params: jsonb not null default '{}'::jsonb
```

Comment:
Setting this as `NOT NULL` makes the migration process really slow, and does not provide any meaningful benefit. This should probably be allowed to be NULL, and then set to an empty object as necessary when processing. Same with the equivalent fields in the other tables.

Reply:
Got it!